src/utils/assign.js

```diff
@@ -0,0 +1,13 @@
+export default function assign(obj, ...sources) {
```

Comment:
Yeah @nfcampos mentioned the polyfill thing, but lodash wasn't already included, so I wrote that. I just checked the [`Object.assign` MDN page](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Object/assign), and it shows an ES5 implementation which I believe is functionally equivalent to mine. We can revert to native `Object.assign`, as the code was already peppered with calls to it before this PR, though.